lms/templates/sudo/third_party_auth_links.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n third_party_auth_links %}
```

Comment:
Can this template and the other one be shared between Studio and LMS by keeping it in the `django_sudo_helpers` app?